go.mod

```diff
@@ -4,133 +4,140 @@ go 1.17
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/coreos/etcd v3.3.13+incompatible
 	github.com/coreos/go-systemd/v22 v22.3.2
 	github.com/gin-gonic/gin v1.6.3
 	github.com/google/martian v2.1.0+incompatible
 	github.com/pkg/errors v0.9.1
-	github.com/rook/rook v1.4.3
-	github.com/spf13/cobra v1.0.0
-	github.com/spf13/viper v1.4.0
+	github.com/rook/rook v1.5.12
+	github.com/spf13/cobra v1.1.3
+	github.com/spf13/viper v1.7.0
 	github.com/stretchr/testify v1.7.0
-	go.uber.org/zap v1.10.0
+	go.etcd.io/etcd/client/v3 v3.5.1
+	go.uber.org/zap v1.19.0
 	go.undefinedlabs.com/scopeagent v0.1.12
-	k8s.io/api v0.22.3
-	k8s.io/apimachinery v0.22.3
+	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/api v0.22.5
+	k8s.io/apimachinery v0.22.5
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/kubernetes v1.17.2
+	k8s.io/kubernetes v1.22.5
+	k8s.io/kubernetes/v121 v121.0.0-00010101000000-000000000000
```

Comment:
Not sure if this is the best practice, but I still need the previous structs to maintain backward compatibility.